contrib/cirrus/win-podman-machine-test.ps1

```diff
@@ -33,5 +33,12 @@ Run-Command ".\bin\windows\podman.exe --version"
 New-Item -ItemType "directory" -Path "$env:AppData\containers"
 Copy-Item -Path pkg\machine\ocipull\policy.json -Destination "$env:AppData\containers"
 
+# Install Perl
+Write-Host "`nInstalling Perl as it is required to use logformatter"
+Run-Command 'choco install --no-progress --confirm --acceptlicense --nocolor StrawberryPerl'
+Write-Host "`nAdd perl to the PATH"
+Set-Item "Env:PATH" "$Env:PATH;C:\Strawberry\perl\bin"
+
+# Run the tests
 Write-Host "`nRunning podman-machine e2e tests"
-Run-Command ".\winmake localmachine"
+Run-Command '.\winmake localmachine | perl .\contrib\cirrus\logformatter $Env:TEST_FLAVOR-podman-windows-rootless-host-sqlite'
```

Comment:
I have no idea how windows deals with this but will the overall command still fail when the test suite fails?
I am a bit paranoid here, can you make a negative test here, i.e. add commit with triggers an intentional failure in the tests to make sure the CI job is still marked as failed in this case.

Reply:
Good idea. Let me try that.

Reply:
If one test fails, the CI job is correctly marked as failed. But the html log file is not generated. I will figure out why.

Reply:
Resolving as the problem has been addressed. The output of the command is always captured in an env variable and processed by `logformatter`. After that an exception is thrown if the (native) command exit code is not zero (same behavior as before).